doc/userGuide.md

```diff
@@ -33,6 +33,111 @@ You can set up your own Jenkins-based test builds using the AdoptOpenJDK openjdk
 
 ![jenkins parameters](/doc/diagrams/jenkinsParameters.jpg)
 
+## How to pass in environment variables
+
+This is the guide on how to pass in environment variables when making builds with Jenkins. For the directions on this, we will be using the example variable below:
+TR_Options='verbose,vlog=testExample1.log'
+There are two ways to do this, and there will be screenshot demonstrations for each step. 
+
+
+#### Method 1: Write it in as part of the playlist file
+
+
+1.	Find the folder that your test is in
+
+ ![test_folder](/doc/diagrams/testFolder.jpg)
+ 
+2.	Open the playlist.xml file
+
+ ![playlist_file](/doc/diagrams/playListFile.jpg)
+ 
+3.	Find the testCaseName matching with the test you want to run
+
+ ![test_case_name](/doc/diagrams/testCaseName.jpg)
+ 
+4.	In the corresponding command section, at the beginning, add the key word `export`, your environment variable, followed by a semicolon
+
+ ![export](/doc/diagrams/commandSection.jpg)
+ 
+5.	Save it, git add, commit, push
+
+ ``` bash
+git add --all
+git commit -m "Added TR_Options as an environment variable in the playlist"
+git push origin env_var
+```
+ 
+6.	Go to the Jenkins page, and open up the Grinders
+
+ ![open_grinders](/doc/diagrams/openGrinders.jpg)
+ 
+7.	Click “Build with Parameters” on the left side of the page, third down from the top
+
+8.	In the ADOPTOPENJDK_REPO section, put in the repository you were working from when you made those changes
+
+ ![repo](/doc/diagrams/repo.jpg)
+ 
+9.	In the ADOPTOPENJDK_BRANCH section, put in the branch you were on
+
+ ![branch](/doc/diagrams/branch.jpg)
+ 
+10.	In the BUILD_LIST and TARGET sections, put in the corresponding information
+
+ ![build_list_target](/doc/diagrams/buildListTarget.jpg)
+ 
+11.	Scroll to the bottom and hit the Build button
+
+ ![build](/doc/diagrams/build.jpg)
+
+
+#### Method 2: Put it in the systemtest.mk file
+
```

Comment:
I am not sure why we put into `systemtest.mk`? This will work if you are going to run system test only. The correct place should be `testEnv.mk`.

Reply:
agree, this is part of our design and about granularity... in playlist for specific test target, in group mk file for application to group only (not a common use case), in testEnv.mk for application to all targets.

Reply:
Where is the testEnv.mk file? I searched through the entire folder and couldn't find it.

Reply:
its part of TestKitGen so after one sets up the test environment on their machine (behind the scenes, test material and scripts are getting fetched from various locations), they would look for it in the TestConfig folder (https://github.com/eclipse/openj9/blob/master/test/TestConfig/testEnv.mk)

somewhat related note, soon we will be moving TestKitGen to its own repo (separate from openj9 and openjdk-tests)...

Reply:
Should I keep in the instructions that keeping the line in the specific tests (systemtest.mk, openjdk.mk, etc) can remain an option, then just add a third option of putting it in testEnv?

Reply:
I think just put the option for testEnv for now, as there is no real use case for the middle option for now and it will increase confusion if we give too many choices.

Reply:
Where would be a good place to put it in testEnv.mk? Right now I have that as the last line, and it isn't doing anything.